src/invidious/routes/before_all.cr

```diff
@@ -100,6 +88,38 @@ module Invidious::Routes::BeforeAll
       end
     end
 
+    # TODO: popular and trending are whitelisted for clients that require these endpoints to be accessible e.g. Clipious
+    # can be removed as soon as those clients can handele these request on private instances
+    unregistered_path_whitelist = {
+      "/login",
+      "/privacy",
+      "/api/v1/stats",
+      "/api/v1/popular",
+      "/api/v1/trending",
+      "/feed/webhook/v1:",
+      "/api/v1/videos/dQw4w9WgXcQ",
```

Comment:
@stonerl  why whitelist this one?

Reply:
`/api/v1/videos/dQw4w9WgXcQ` is for the healthcheck in docker I think
`/feed/webhook/v1:` for keeping pubsub working